syntax = "proto3";

package hello;

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string greet = 1;
}

service Hello {
  rpc SayHello(HelloRequest) returns(HelloResponse) {
  }
  rpc BatchHello(stream HelloRequest) returns(HelloResponse) {
  }
}
